index.html

```diff
@@ -508,48 +552,58 @@ <h2 class="sign-up-block__title title">Get Start New Job</h2>
         </div>
       </section>
       <!-- Third section -->
-      <section class="third-section">
-        <div class="third-section__background">
-          <div class="container third-section__container">
+      <section class="post-job">
+        <div class="post-job__background">
+          <div class="container post-job__container">
             <!-- Third section left wrapper -->
-            <div class="third-section__left-wrapper">
+
+            <div class="post-job__video-block">
               <img
-                class="third-section__img"
-                src="./src/img/third-section-img.png"
+                class="post-job__img"
+                src="./src/img/post-job-img.png"
                 alt=""
               />
-              <a href="#" class="play-button"></a>
+
+              <button class="play-button">
+                <img src="./src/icons/play-button.svg" alt="play button" />
+              </button>
             </div>
             <!-- Third section right wrapper -->
-            <div class="third-section__right-wrapper">
-              <h2 class="third-section__title title">
-                Post a Job for UX Designer
-              </h2>
-              <p class="third-section__subtitle">
+            <div class="post-job__info">
+              <h2 class="post-job__title title">Post a Job for UX Designer</h2>
+              <p class="post-job__subtitle">
                 Create an account so you can get job information that suits you,
                 and you can apply the salary your demand.
               </p>
-              <a class="third-section__buttons buttons" href="#">
+              <button class="post-job__button button" href="#">
                 Post a Job
-              </a>
+              </button>
             </div>
           </div>
         </div>
       </section>
 
-      <!-- Fourth-section -->
-      <section class="fourth-section">
-        <div class="container fourth-section__container">
-          <h2 class="fourth-section__title title">
-            Great Feedback From Candidates
-          </h2>
-          <div class="fourth-section__feedback-wrapper">
+      <!-- feedback -->
+      <section class="feedback">
+        <div class="feedback__container">
```

Comment:
```suggestion
      <section class="feedback">
```
You don't need an extra wrapper. Move styles to `feedback`